requirements.txt

```diff
@@ -0,0 +1,6 @@
+Click==7.0
+Flask==1.0.3
+itsdangerous==1.1.0
+Jinja2==2.10.1
+MarkupSafe==1.1.1
+Werkzeug==0.15.4
```

Comment:
How many of these are directly required by the application and how many are flask dependencies?  I may be wrong, but if they're just flask dependencies then they should be installed automatically.  @samvarankashyap you may know more about this than I do

Reply:
@mansikulkarni96 
I agree with @14rcole 
according to flask project: 
https://github.com/pallets/flask/blob/master/setup.py
```
        "Werkzeug>=0.15",
        "Jinja2>=2.10.1",
        "itsdangerous>=0.24",
        "click>=5.1",
```
are already included as part of flask installation. 
Thus making them unnecessary for the current project if the flask is included. 
You can remove them from the current requirements file. 
@14rcole Thanks for pointing it out.

Reply:
I will remove them all, thanks @14rcole  and @samvarankashyap